force_bdss/core/verifier.py

```diff
@@ -100,23 +112,27 @@ def _check_data_source(data_source_model):
     if len(input_slots) != len(data_source_model.input_slot_info):
         errors.append(VerifierError(
             subject=data_source_model,
-            error="Missing input slot name assignment"))
+            error="Missing input slot name assignment "
+                  "in layer {}".format(layer_no)))
```

Comment:
you can also put the data source name here for additional information. You get it from the factory